README.md

```diff
@@ -49,20 +49,18 @@ And a quick crash course on inference quantization to help parse the above table
 
 Sparsifying your model is also a 1 liner that should work on any model with an `nn.Linear`. We find that sparsity works best on compute bound models like SAM, specifically the MLP layers.
 ```python
-from torchao.sparsity import sparsify
-from torch.sparse import to_sparse_semi_structured
+from torchao.sparsity import sparsify, semi_sparse_weight()
 
-m = sparsify(m, to_sparse_semi_structured)
+m = sparsify_(m, semi_sparse_weight())
```

Comment:
does `semi_sparse_weight` have to talk about dtype as well?

Reply:
it will work for bf16, fp16, and fp32, so i don't think specifying the dtype makes sense. Maybe `dense_activation_semi_sparse_weight` to keep it consistent?

Reply:
I see, then it's fine. we have `int4_weight_only()` as well so I feel it's fine that we don't mention activation

(we could remove `only` as well)